lib/helpers.rb

```diff
@@ -15,8 +15,12 @@ def thread
     @thread ||= CommentThread.find(params[:thread_id])
   end
 
-  def comment
-    @comment ||= Comment.find(params[:comment_id])
+  def comment(comment_id=nil, sync_child_count=false)
```

Comment:
See comment above.  I don't see where this is called with comment_id param, so I'm not sure why it was added.